static/src/js/containers/UrlQueryContainer/UrlQueryContainer.jsx

```diff
@@ -38,15 +37,14 @@ export const mapStateToProps = (state) => ({
   keywordSearch: state.query.collection.keyword,
   lineSearch: state.query.collection.spatial.line,
   location: state.router.location,
-  mapPreferences: getMapPreferences(state),
   onlyEosdisCollections: state.query.collection.onlyEosdisCollections,
   overrideTemporalSearch: state.query.collection.overrideTemporal,
   pathname: state.router.location.pathname,
   pointSearch: state.query.collection.spatial.point,
   polygonSearch: state.query.collection.spatial.polygon,
   project: state.project,
   query: state.query,
-  paramCollectionSortKey: getCollectionSortKeyParameter(state),
+  paramCollectionSortKey: state.query.collection.paramCollectionSortKey,
```

Comment:
This is where the logic came from that you added into url.js. The logic needs to be removed from url.js, it is now handling that parameter differently than all the other parameters. 

But I don't know if I have an answer to how I think this should work yet (we should pair again tomorrow on the selector's stuff).